app/viewModels/ViewMovement.scala

```diff
@@ -34,22 +34,35 @@ object ViewMovement {
       arrival.updated.toLocalTime,
       arrival.movementReferenceNumber,
       status(arrival),
-      actions(arrival.movementReferenceNumber, arrival.status)
+      actions(arrival, arrival.status)
     )
 
   private def status(arrival: Arrival)(implicit messages: Messages, frontendAppConfig: FrontendAppConfig) = arrival.status match {
     case "UnloadingPermission" => Messages("movement.status.unloadingPermission")
+    case "ArrivalRejected"     => Messages("movement.status.arrivalRejected")
     case "ArrivalSubmitted"    => Messages("movement.status.arrivalSubmitted")
+    case "GoodsReleased"       => Messages("movement.status.goodsReleased")
     case _                     => arrival.status
   }
 
-  private def actions(mrn: String, status: String)(implicit messages: Messages, frontendAppConfig: FrontendAppConfig): Seq[ViewMovementAction] = status match {
-    case "UnloadingPermission" =>
-      Seq(
-        ViewMovementAction(frontendAppConfig.declareUnloadingRemarksUrl(mrn), Messages("viewArrivalNotifications.table.action.unloadingRemarks"))
-      )
-    case _ => Nil
-  }
+  private def actions(arrival: Arrival, status: String)(implicit messages: Messages, frontendAppConfig: FrontendAppConfig): Seq[ViewMovementAction] =
```

Comment:
The status and actions methods are looking like they need splitting out maybe. Both methods are doing a match on status and the model is probably doing too many things now. It doesn't need doing in this story but if anyone wants to have a think about how it could be split out that would be great